src/main/java/mcjty/theoneprobe/apiimpl/providers/DefaultProbeInfoProvider.java

```diff
@@ -188,10 +190,15 @@ private void showRF(IProbeInfo probeInfo, World world, BlockPos pos) {
             if (handler != null) {
                 addRFInfo(probeInfo, config, handler.getEnergyStored(), handler.getMaxEnergyStored());
             }
+        } else if (Loader.isModLoaded("tesla") && te != null && te.hasCapability(TeslaCapabilities.CAPABILITY_HOLDER, null)) {
```

Comment:
It won't come to that statement if Tesla isn't loaded.
Loader.isModLoaded("tesla") will return false if it's not loaded, which results in skipping the rest of the if statement.